app/lib/sms-games/controllers/SGCompetitiveStoryController.js

```diff
@@ -174,10 +174,11 @@ SGCompetitiveStoryController.prototype.createGame = function(request, response)
 
     // We opt users into these initial opt in paths only if the game type is NOT solo. 
     if (self.createdGameDoc.game_type !== 'solo') {
+      var gameConfig = app.getConfig('competitive_stories_config', self.createdGameDoc.story_id)
```

Comment:
I think the `gameConfig` here clashes with the `gameConfig` defined up on line 68. Unless that's the intent, straightforward fix would be just naming this var something different.